index.html

```diff
@@ -141,7 +141,7 @@ <h2 class="card-title">Profesional Art Printing Data</h2>
       </section>
       <section class="about">
         <div class="about-me">
-          <h2 class="about-title">About me</h2>
+          <h2 class="about-title">About</h2>
           <p class="about-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
           <button class="resume">Get my resume</button>
         </div>
```

Comment:
Use the <button> tag outside the text container. We recommend that for proper CSS